crates/net/network/src/builder.rs

```diff
@@ -49,7 +51,9 @@ impl<C, Tx, Eth> NetworkBuilder<C, Tx, Eth> {
         client: Client,
     ) -> NetworkBuilder<C, Tx, EthRequestHandler<Client>> {
         let NetworkBuilder { mut network, transactions, .. } = self;
-        let (tx, rx) = mpsc::unbounded_channel();
+        // We use twice the maximum number of available slots, if all slots are occupied
+        // This is the same as session_event_buffer size in [`SessionsConfig`]
+        let (tx, rx) = mpsc::channel((DEFAULT_MAX_PEERS_OUTBOUND + DEFAULT_MAX_PEERS_INBOUND) * 2);
```

Comment:
hmm, I thought about this again, and this differs from the p2p event buffer because there's only a single sender/receiver pair.

this also does not respect the actual configured values for the session config.

I think we can make the channel capacity 256
this should be more than enough, since this sits already behind a bounded event channel.

256 requests with malicious 10MB body requests is 2.6GB however this should be absorbed.